.gitlab/condition.yml

```diff
@@ -0,0 +1,36 @@
+.pr_condition:
+  only:
+    variables:
+      - $RUN_CI_TAG
+  except:
+    refs:
+      - develop
+      - master
+      - tags
+  dependencies: []
+
+.full_test_condition:
+  only:
+      variables:
+        - $RUN_CI_TAG && $STATUS_CONTEXT == "full"
+        - $RUN_CI_TAG && ($CI_COMMIT_BRANCH == "master" || $CI_COMMIT_BRANCH == "develop")
+        - $RUN_CI_TAG && $CI_COMMIT_TAG
+  dependencies: []
+
+.quick_test_condition:
+  only:
+    variables:
+      - $RUN_CI_TAG && $STATUS_CONTEXT == null
+  dependencies: []
```

Comment:
So the quick test are not ran when running the full pipeline (with the label set)?

Reply:
quick test in the quick pipeline. the full pipeline will only run the remaining jobs.
the pr always contains both pipelines with the label set.

Reply:
I see I understand now, thanks.